style.css

```diff
@@ -1208,55 +1215,181 @@ button :disabled {
     margin-right: 10%;
   }
 
-  .hamburger {
-    display: block;
-    margin-top: 25px;
+  .intro_item {
+    margin-top: 190px;
+  }
+
+  .multi_post {
+    margin-top: 120px;
+    width: 100%;
+  }
+
+  .form {
+    flex-direction: column;
+    align-items: flex-start;
+    width: 90%;
+    margin: 120px 5% 0 5%;
+  }
+
+  .form_field_item4 {
+    align-self: center;
+    justify-content: center;
+  }
+
+  .action {
+    display: flex;
+  }
+
+  .form_item {
+    align-self: center;
+  }
+
+  .form_item1 {
+    width: 100%;
+  }
+
+  .recent_works {
+    margin-bottom: 20px;
+  }
+
+  .main_container p {
+    width: 327px;
+    height: 148px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .form_item1 p {
+    text-align: center;
+  }
+
+  .about_block h3 {
+    margin-top: 60px;
+    margin-left: 1px;
+  }
+
+  #about p {
+    width: 327px;
+    height: 148px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    margin-left: -25px;
+  }
+}
+
+@media screen and (max-width: 500px) {
```

Comment:
- [OPTIONAL] The maximum mobile screen width is `768px`. up to 768px is considered mobile (and tab) screen size. above that is all desktop screen sizes. for that reason we should apply mobile menu in our navbar up to 768px. the problem in your design is that from `500px` to `768px` your navbar dont have a mobile menu because your breaking points are at 500px. kindly consider changing your breakpoints at `768px` and try fixing your navbar which breaks from 500px to 768px.
your design between 768px and 500px
![qwe](https://user-images.githubusercontent.com/96683843/224124767-948c3455-f5f3-4101-9986-fcdd3b7c9b3f.jpg)